mpegdash/nodes.py

```diff
@@ -335,6 +335,39 @@ def write(self, xmlnode):
         write_attr_value(xmlnode, 'ns2:default_KID', self.key_id)
 
 
+class PSSH(XMLNode):
+    def __init__(self):
+        self.pssh = None
+
+    def parse(self, xmlnode):
+        self.pssh = parse_node_value(xmlnode, str)
+
+    def write(self, xmlnode):
+        write_node_value(xmlnode, self.pssh)
+
+
+class ContentProtectionComponent(XMLNode):
+    def __init__(self):
+        self.scheme_id_uri = ""  # xs:anyURI (required)
+        self.value = None  # xs:string
+        self.id = None  # xs:string
+        self.key_id = None  # xs:string
+        self.pssh = None
+
+    def parse(self, xmlnode):
+        self.scheme_id_uri = parse_attr_value(xmlnode, "schemeIdUri", str)
+        self.value = parse_attr_value(xmlnode, "value", str)
+        self.id = parse_attr_value(xmlnode, "id", str)
+        self.key_id = parse_attr_value(xmlnode, "ns2:default_KID", str)
+        self.pssh = parse_child_nodes(xmlnode, "pssh", PSSH)
+
+    def write(self, xmlnode):
+        write_attr_value(xmlnode, "schemeIdUri", self.scheme_id_uri)
+        write_attr_value(xmlnode, "value", self.value)
+        write_attr_value(xmlnode, "id", self.id)
+        write_attr_value(xmlnode, "ns2:default_KID", self.key_id)
+        write_child_node(xmlnode, "cenc:pssh", self.pssh)
```

Comment:
parsing as `pssh`, but writing as `cenc:pssh`. Is this ideal? It's also only accepting ns2:default_KID, but more namespaces use ns2, including `cenc`.

Reply:
It is because `parse_child_nodes` checks tag name using `localName` property and `localName` of "cenc:pssh" is "pssh". But now I have modified it to check either `localName` or `nodeName`.